config/global.ini.php

```diff
@@ -910,12 +910,15 @@
 [Deletelogs]
 ; delete_logs_enable - enable (1) or disable (0) delete log feature. Make sure that all archives for the given period have been processed (setup a cronjob!),
 ; otherwise you may lose tracking data.
-; delete_logs_schedule_lowest_interval - lowest possible interval between two table deletes (in days, 1|7|30). Default: 7.
+; delete_logs_schedule_lowest_interval - lowest possible interval between two table deletes, for table log_visit (in days, 1|7|30). Default: 7.
 ; delete_logs_older_than - delete data older than XX (days). Default: 180
+; delete_logs_unused_actions_schedule_lowest_interval - lowest possible interval between two table deletes, for table log_action (in days, 1|7|30). Default: 30.
+; *_max_rows_per_query - can be increased for large sites to speed up delete processes
```

Comment:
btw... the longer you set the value, the longer the delete will take, the longer the table might be locked (which will cause incoming tracking requests to pile up since they can't finish and are waiting for the lock). It's probably not an issue with InnoDB since it should have row locking though.

Reply:
btw not seeing this one used anywhere, are you still working on the PR?

Reply:
I see now, you're only adding it for documentation purposes 👍

Reply:
Yes :) I'll add some more docs as well as I think it is valuable to have there